### DISCUSSION TITLE
Get more details

### DISCUSSION THREAD

**Discussion by tomyems** [UKHomeOffice/design-system#480] [General]
To discuss improvements and suggestions for the [get more details](https://design.homeoffice.gov.uk/patterns/get-more-details) pattern.

**Comment by mash0904** [comment]
At GDS I'm exploring how we might provide users with contextual help and would like to take a closer look at this approach.

Are you able to share a working version?

**Reply by tomyems** [reply]
Ah this pattern has been there for some time, let me see if I can get in touch with the designers on the services using the pattern

**Comment by tomyems** [comment]
Is this pattern actually being used in the Home Office?
